model/Core/Properties/requirementContext.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# requirementContext
+
+## Summary
+
+Context of the requirement relating it to a project and its structure.
```

Comment:
My comment about `contextOrigin` strongly applies here as well: https://github.com/spdx/spdx-3-model/pull/1061/files#r2324881244.

Would it not make sense to identify the context for an enclosing document and then have all requirements inherit this property by lookups. This makes requirements more portable and reduces repetition.

What defines the context is there this requirement appears, e.g., in which document, section, subsection, etc.